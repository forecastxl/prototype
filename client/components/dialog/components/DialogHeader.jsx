import React, { PropTypes } from 'react'
import { CardHeader } from 'material-ui/Card'
import muiThemeable from 'material-ui/styles/muiThemeable'

export function DialogHeader({ muiTheme, ...rest }) {
  return (
    <CardHeader style={{ background: muiTheme.palette.primary1Color }} {...rest} />
  )
}

DialogHeader.propTypes = {
  muiTheme: PropTypes.object.isRequired
}

export default muiThemeable()(DialogHeader)
