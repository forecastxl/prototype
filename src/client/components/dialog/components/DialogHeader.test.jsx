import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import { DialogHeader } from './DialogHeader'

describe('<DialogHeader />', () => {
  it('renders correctly', () => {
    const theme = {
      palette: {
        primary1Color: 'blue'
      }
    }
    const wrapper = shallow(<DialogHeader muiTheme={theme} />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
