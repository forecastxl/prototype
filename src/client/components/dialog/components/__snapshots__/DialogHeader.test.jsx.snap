// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DialogHeader /> renders correctly 1`] = `
<CardHeader
  avatar={null}
  style={
    Object {
      "background": "blue",
    }
  }
/>
`;
